images/airflow/2.10.1/BillOfMaterials/2.10.1-dev/Python-Packages-BOM.txt

```diff
@@ -6162,10 +6162,10 @@ Copyright 2013-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 /usr/local/airflow/.local/lib/python3.11/site-packages/boto3-1.35.7.dist-info/NOTICE
 
 boto3-stubs
-1.35.21
+1.35.13.post1
```

Comment:
The versions are going backwards by two months. I believe you will need to use run.sh for the two versions in order to generate these files with the latest versions. 

OR, maybe just remove them and some other commit made by someone else can take care of updating these files.

Reply:
This was the result of the lastest run for me. Removing the Bill of materials as discussed offline as it can be updated in another change